functests/utils/consts.go

```diff
@@ -24,7 +24,15 @@ var NodesSelector string
 // ProfileNotFound is true when discovery mode is enabled and no valid profile was found
 var ProfileNotFound bool
 
+// PerformanceOperatorNamespace contains the name of the performance operator namespace
+var PerformanceOperatorNamespace string
```

Comment:
I feel like namespaces.PerformanceOperator would be a better fit for this variable.
Also, (but it's a matter of taste) I'd like to see the variable initialized with the default value and overridden if needed

Reply:
Right, but this calls for a bigger refactoring than what I did here.
... Which is actually a great idea. Let's see what I can do.

Reply:
Not that big.
The namespace package is already there, you can just move the variable and the initialization.